frontend/components/home.jsx

```diff
@@ -57,56 +79,260 @@ class Home extends React.Component {
     this.createInterval();
   }
 
+  playNextSongDifferent() {
+    // console.log('Next song is different');
+    if (this.currentTrack === "one") {
+      this.currentTrack = "two";
+      this.setState({
```

Comment:
Let's create a method for this and pass in a flag of which sound component is active and do basic logic in there.